bankwire.php

```diff
@@ -298,15 +343,27 @@ public function renderForm()
             'id_language' => $this->context->language->id
         );
 
-        return $helper->generateForm(array($fields_form));
+        return $helper->generateForm(array($fields_form, $fields_form_customization));
     }
 
     public function getConfigFieldsValues()
     {
+        $custom_text = array();
+        $languages = Language::getLanguages(false);
+        foreach ($languages as $lang) {
+            $custom_text[$lang['id_lang']] = Tools::getValue(
+                'BANK_WIRE_CUSTOM_TEXT_'.$lang['id_lang'],
+                Configuration::get('BANK_WIRE_CUSTOM_TEXT', $lang['id_lang']
+                )
```

Comment:
https://github.com/PrestaShop/ps_emailsubscription/commit/ecffc8e2c1dd8763d8d7d313a0ecffd7dfb2dff6#diff-cfb1aa2304713070e111aaa57e5c8ef5R1031

# Do as I say, not as I do :laughing: